libevmasm/GasMeter.h

```diff
@@ -67,16 +67,29 @@ namespace GasCosts
 	}
 	static unsigned const keccak256Gas = 30;
 	static unsigned const keccak256WordGas = 6;
+	static unsigned const coldSloadCost = 2100;
+	static unsigned const coldAccountAccessCost = 2600;
```

Comment:
I think you should update `extCodeGas()`, `balanceGas()`, `callGas()` and maybe some others to charge the extra `coldAccountAccessCost` on Berlin.